msteams-platform/graph-api/rsc/resource-specific-consent.md

```diff
@@ -101,6 +101,7 @@ The following table provides RSC application permissions for a team and their ap
 |`Owner.Read.Group`|Read this group's owners.| NA | Supported |
 |`TeamsActivity.Send.Group`|Send activity feed notifications to users in this team. | NA | Supported |
 |`TeamsAppInstallation.Read.Group`|Read the apps that are installed in this team.| NA | Supported |
+|`TeamsAppInstallation.ReadWriteAndConsentSelfForChat.All`|Allows a Teams app to read, install, upgrade, and uninstall itself for any chat, without a signed-in user, and manage its permission grants for accessing those specific chats data.| NA | Supported |
```

Comment:
This is not an RSC permission. We should not be adding it here.

Reply:
Hi @AkJo: Request you to let me know where exactly to be added.